src/qt/voting/pollcard.cpp

```diff
@@ -24,6 +24,20 @@ PollCard::PollCard(const PollItem& poll_item, QWidget* parent)
     ui->totalWeightLabel->setText(QString::number(poll_item.m_total_weight));
     ui->activeVoteWeightLabel->setText(QString::number(poll_item.m_active_weight));
     ui->votePercentAVWLabel->setText(QString::number(poll_item.m_vote_percent_AVW, 'f', 4) + '\%');
+
+    // See voting/fwd.h, the enum class PollWeightType
+    if (!(poll_item.m_weight_type == 2 || poll_item.m_weight_type == 3)) {
+        ui->balanceLabel->hide();
+    }
+
+    if (!(poll_item.m_weight_type == 1 || poll_item.m_weight_type == 3)) {
+        ui->magnitudeLabel->hide();
+    }
+
+    if (poll_item.m_validated.toString() == QString{} || !poll_item.m_validated.toBool()) {
```

Comment:
```suggestion
    if (poll_item.m_validated.toString() == QString{}) {
```
Would not want to hide it if the poll did not pass.

Reply:
No, it is exactly the opposite. The validation tag is to indicate that a poll has the requisite AVW to be validated. If it is blank (meaning the poll version doesn't support it i.e. < v3), or validation is false, then validated should be hidden. It is displayed once the AVW % gets to the rule value for the poll type in context.

Reply:
Good point, that does make sense for running polls. Although if the poll did finish, it can't get anymore AVW so showing that it did fail makes sense imo.
```suggestion
    if (poll_item.m_validated.toString() == QString{} || (!poll_item.m_finished && !poll_item.m_validated.toBool())) {
```

Reply:
So should we have an INVALID in red?

Reply:
Ok I just implemented this with the invalid tag added and tested it with the four testnet polls that are in scope of the display by temporarily removing the v2 version requirement from PollResult::BuildFor and also changing the SURVEY min vote weight % AVW to 30 in Poll::PollTypeRules POLL_TYPE_RULES[]. (This is so the different AVW %'s would show different validation status.